sdk/storage/storage-blob-changefeed/samples/README.md

```diff
@@ -0,0 +1,83 @@
+# Getting started with samples
+
+## Install the library
+
+Copy the sample file you want to a folder of your choice and run the following command in your samples folder to install the npm package for Azure Storage Blob library.
+
+```bash
+npm install @azure/storage-blob-changefeed
+```
+
+Or, in case you have cloned the repo and want to utilize the published npm package instead of source code, run the following commands in the `samples` folder to initialize an npm project and to install the storage-blob-changefeed package.
+
+```bash
+npm init -y
+npm install @azure/storage-blob-changefeed
+```
+
+## Get account credentials of your storage account
+
+- In the [Azure Portal](https://portal.azure.com), go to **Dashboard > Storage > _your-storage-account_**.
+- Note down the "AccountName", "AccountKey" obtained at **Access keys** and "AccountSAS" from **Shared access signature** under **Settings** tab.
+  Before running any of the samples, update with the credentials you have noted down above.
+
+### Authenticating with Azure Active Directory
+
+If you have [registered an application](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app) with an Azure Active Directory tenant, you can [assign it to an RBAC role](https://docs.microsoft.com/en-us/azure/storage/common/storage-auth-aad) in your Azure Storage account. This enables you to use the Azure.Identity library to authenticate with Azure Storage as shown in the [azureAdAuth.ts sample](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/storage-blob/samples/typescript/src/azureAdAuth.ts).
+
+## JavaScript sample
+
+Copy the sample to your samples folder and use `node` to run it.
+
+```bash
+node sample.js
+```
+
+## TypeScript sample
+
+If you don't have TypeScript installed, then use `npm` to install it first.
+
+```bash
+npm install -g typescript
+```
+
+One way to run TypeScript samples is to use `ts-node`. To install `ts-node`, run the below in your sample folder
+
+```bash
+npm install -g ts-node
+```
+
+Copy the sample to your samples folder and use `ts-node` to run it.
+
+```bash
+ts-node sample.ts
+```
+
+## Browser sample
```

Comment:
Maybe remove browser sample section for now as it's not yet supported.

Reply:
Fixed in the change feed PR: https://github.com/Azure/azure-sdk-for-js/pull/10827/commits/367dfb0fb7d13df81ca6308f0f1e58033b2eb794